src/app/groups/group-member-contribution-assigner/group-member-contribution-assigner.tpl.html

```diff
@@ -11,7 +11,7 @@
       <tr ng-repeat="member in team.members | orderBy:memberSortOrder:reverse">
         <td>{{member.student_name}}</td>
         <td class="text-left">
-          <grade-icon grade="member.target_grade"></grade-icon>
+          <grade-icon [index]="member.target_grade"></grade-icon>
```

Comment:
what is index here?

Reply:
Shouldn't it be grade?

Reply:
Hi @jakerenzella, thanks for the feedback! My reason for separating grade into grade and index is that some parent components are giving the component a grade value, such as "Pass", and others are passing the index value of the grade, such as "1". Perhaps a better option would be making sure all parent components pass only the grade.

Reply:
Ah, then if the type is an enumeration of a f p c d hd we can map the index automatically 👍

Reply:
It's also worth noting that `F` was originally mapped to `-1` so we need to make sure it does not cause regression by moving to `0` base system.